rama-net/src/client/conn.rs

```diff
@@ -1,38 +1,30 @@
 use rama_core::{Service, error::BoxError, extensions::ExtensionsMut, service::BoxService};
 use std::fmt;
 
+#[derive(Clone)]
```

Comment:
not something that we want to or should enforce, I prefer the manual clone impl here